packages/lib/person/service.ts

```diff
@@ -185,17 +192,24 @@ export const createPerson = async (environmentId: string): Promise<TPerson> => {
   }
 };
 
-export const deletePerson = async (personId: string): Promise<void> => {
+export const deletePerson = async (personId: string): Promise<TPerson | null> => {
   validateInputs([personId, ZId]);
+
   try {
-    await prisma.person.delete({
+    const personPrisma = await prisma.person.delete({
```

Comment:
sometimes it's `person`/`people`, sometimes it's `personPrisma`/`peoplePrisma`

Reply:
Renamed every personPrisma to person and then when `transformPrismaPerson` is called, I name the variable to `transformedPerson`

What do you think?

```
    const person = await prisma.person.create({
      data: {
        environment: {
          connect: {
            id: environmentId,
          },
        },
      },
      select: selectPerson,
    });

    const transformedPerson = transformPrismaPerson(person);
```

Reply:
Done